Lib9c/Action/TransferAsset.cs

```diff
@@ -9,21 +9,32 @@
 using System.Linq;
 using System.Runtime.Serialization;
 using Lib9c.Abstractions;
+using Nekoyume.Helper;
 using Nekoyume.Model;
 using Serilog;
 
 namespace Nekoyume.Action
 {
     /// <summary>
     /// Hard forked at https://github.com/planetarium/lib9c/pull/636
-    /// Updated at https://github.com/planetarium/lib9c/pull/957
+    /// Updated at https://github.com/planetarium/lib9c/pull/1718
     /// </summary>
     [Serializable]
     [ActionType("transfer_asset3")]
     public class TransferAsset : ActionBase, ISerializable, ITransferAsset, ITransferAssetV1
     {
         private const int MemoMaxLength = 80;
 
+        public const long WhiteListAvailableIndex = 6_200_000L;
```

Comment:
```suggestion
        public const long CrystalTransferringRestrictionStartIndex = 6_200_000L;
```

if we feel `CrystalTransferringRestriction` is too verbose, leveraging an external NCIP number (if exists) would be good too. cc @jaeho0103